sdk/tests/invariant.test.ts

```diff
@@ -449,6 +451,36 @@ describe('invariant', async () => {
       }
       await positionEquals(position, expectedPosition)
     })
+    it('calculate token amounts from position liquidity', async () => {
+      const position = await invariant.getPosition(account, account.address, 0n)
+      const pool = await invariant.getPool(
+        account,
+        token0.contract.address.toString(),
+        token1.contract.address.toString(),
+        feeTier
+      )
+
+      const providedAmount = 500n
+      const expectedYAmount = getLiquidityByX(
```

Comment:
I don't like this syntax, I think it is much less readable:
```TypeScript
const expectedYAmount = getLiquidityByX(...).amount
```
IMO this one is better
```TypeScript
const { amount: expectedYAmount } = getLiquidityByX(...)
```
1. It's easy to miss reference to field.
2. harder to take another result parameter